content/docs/2.16/scalers/selenium-grid-scaler.md

```diff
@@ -30,12 +30,15 @@ triggers:
 **Parameter list:**
 
 - `url` - Graphql url of your Selenium Grid. Refer to the Selenium Grid's documentation [here](https://www.selenium.dev/documentation/en/grid/grid_4/graphql_support/) to for more info.
+- `username` - Username for basic authentication in GraphQL endpoint instead of embedding in the URL. (Optional)
+- `password` - Password for basic authentication in GraphQL endpoint instead of embedding in the URL. (Optional)
```

Comment:
Let's decide on this in the code first: https://github.com/kedacore/keda/pull/6169#discussion_r1785136083